docs/docs/pulsar-specific/REPOS.md

```diff
@@ -0,0 +1,209 @@
+# Repos
```

Comment:
This should be broken up into a specific section about this, especially so we can include easily contents of it from it's specific folders.

Reply:
What should I break up? Do you mean categorizing inside a new section?